contributor/03-contributor-ethos-article.md

```diff
@@ -21,56 +21,51 @@ knocking or ringing the door bell even if the door is open. Likewise in InnerSou
-your proposed solution. Spending some time on an initial design instead of
-jumping right into the implementation here saves contributors from having to
-re-do a lot of their work. Sharing progress early - even when it's not finished
-yet - helps mentoring the contributor towards a better solution. Much like
+What about those lovely summer parties that you went to? 
+There is usually some planning ahead of time to choose the right date and time, to
+prepare enough food, or to have it contributed by guests. The same happens for
+bigger changes in InnerSource projects: a project will likely ask you to submit 
+an issue describing your need and your proposed solution before making a large modification. 
+Spending time on initial design instead of
+jumping right into the implementation saves contributors from having to
+redo a lot of their work. Sharing progress early - even when it's not finished
+yet - helps the host team to mentor the Contributor towards a better solution. Much like
 [Yonik's law of unifinished
 patches](https://cwiki.apache.org/confluence/display/solr/HowToContribute)
 explains: "A half-baked patch in Jira, with no documentation, no tests
 and no backwards compatibility is better than no patch at all."
 
 Does that imply that InnerSource projects place no value on face to face
 communication? Not quite: there is value in meeting participants face to face.
-Remember that all written communication lacks a lot of bandwidth compared with
+Remember that all written communication lacks a lot of bandwidth compared to
 meeting in person: there are no gestures, no mimics, not even the tone of voice
-that otherwise help with understanding. In particular for inter-personal
-challenges, resolving conflict and misunderstandings meeting in person has
-strong value. However for InnerSource projects the goal should be that
-communication about project decisions be kept in writing so that others can
-follow along and influence the project, so that even years later it's possible
+to help with understanding. In-person meetings are particularly valuable for 
+interpersonal challenges, resolving conflicts and misunderstanding. 
+However, communication about project decisions should be kept in writing, so that others can
+follow along and influence the project, and even years later it's possible
 to trace why a certain decision was made.
 
-Here's my general rule of thumb: feel free to meet over coffee, often that helps
-with building a stronger team. Remember though that when the team is split into
-multiple physical locations (for me that includes teams that are spread across
-mutliple floors), there is a good chance that this encourages the team to split
-into multiple silos. Do make sure though that all decisions are made in a
+Here's my general rule of thumb: feel free to meet over coffee. Often that helps 
+build a stronger team. Do make sure though that all decisions are made in a
```

Comment:
I did like the example here, since I've seen that effect myself repeatedly if it is not counteracted e.g. using Slack and/or regular contact e.g. over a coffee or the likes.

Reply:
How about referencing multiple locations but letting readers decide what that means for them?

```suggestion
build a stronger team, especially when the team is split into multiple physical locations. Do make sure though that all decisions are made in a
```